main.py

```
@@ -1,45 +1,84 @@
+                    logger.info(f"암표 의심 군집 {len(suspicious_clusters)}건이 성공적으로 적발되었습니다!")
+                else:
+                    logger.info("탐지된 이상 결제 군집이 없습니다.")
+            finally:
+                fraud_detector.close()
+
+        logger.info("=== 모든 파이프라인 처리가 성공적으로 종료되었습니다 ===")
 
-    logger.info(" 파이프라인 처리가 성공적으로 종료되었습니다. ")
+    except Neo4jError as db_err:
+        logger.error(f"Neo4j 데이터베이스 처리 중 에러 발생: {db_err}")
+    except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)